src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/ApplicationDesignerEditorFactory.vb

```diff
@@ -179,7 +180,18 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
         Implements IVsEditorFactory.CreateEditorInstance
 
             ' If we're using the new project properties editor, delegate to its editor factory
-            If UseNewEditor(Hierarchy) Then
+            Dim shouldUseNewEditor As Boolean = UseNewEditor(Hierarchy)
+
+            Dim TelemetryEventRootPath As String = "vs/projectsystem/propertiespages/"
+            Dim TelemetryPropertyPrefix As String = "vs.projectsystem.propertiespages."
+
+            Dim telemetryEvent As TelemetryEvent = New TelemetryEvent(TelemetryEventRootPath + "CreateLegacyEditor")
+            telemetryEvent.Properties(TelemetryPropertyPrefix + "CreateLegacyEditor.UseNewEditor") = shouldUseNewEditor
+            telemetryEvent.Properties(TelemetryPropertyPrefix + "CreateLegacyEditor.FileName") = New TelemetryPiiProperty(FileName)
+            telemetryEvent.Properties(TelemetryPropertyPrefix + "CreateLegacyEditor.PhysicalView") = PhysicalView
+            TelemetryService.DefaultSession.PostEvent(telemetryEvent)
```

Comment:
Consider moving this to `Microsoft.VisualStudio.Editors.AppDesCommon.Utils.TelemetryLogger`.

Reply:
I've made changes the TelemetryLogger in [my PR](https://github.com/dotnet/project-system/pull/6969), so there might be conflicts depending who merges first.